test/users/ferguson/fielderator-ref.chpl

```diff
@@ -31,9 +31,6 @@ proc myproc(type t, ref m) {
   assert(__primitive("field value by num", m, 1) == 3);
   assert(__primitive("field value by num", m, 2) == "hi");
   assert(realEqual(__primitive("field value by num", m, 3), 17.23));
-  assert(__primitive("field value by name", m, "x") == 3);
-  assert(__primitive("field value by name", m, "y") == "hi");
-  assert(realEqual(__primitive("field value by name", m, "z"), 17.23));
```

Comment:
Since we're not really invested in the primitives anymore (i.e., if they changed name/arguments, we wouldn't necessarily care if the user interface kept working), should we rewrite this test and fielderator.chpl to use the public interfaces for additional exposure/coverage?